client/components/Special/ListIgnored.vue

```diff
@@ -9,12 +10,18 @@
 		<tbody>
 			<tr v-for="user in channel.data" :key="user.hostmask">
 				<td class="hostmask"><ParsedMessage :network="network" :text="user.hostmask" /></td>
-				<td class="when">{{ localetime(user.when) }}</td>
+				<td class="when">{{ getLocaletime(user.when) }}</td>
```

Comment:
Not sure this is a necessary name change

Reply:
I changed it because we import `localetime` and also use it as method name. Technically this works fine, but to make it easier to distinguish, a different name would be better imo. :)